.github/workflows/playwright.yml

```
@@ -0,0 +1,32 @@
+name: 🎭 Playwright
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.4
+        name: Install pnpm
+        with:
+          version: 7
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: pnpm
+      - name: Install dependencies
+        run: pnpm install
+      - name: Install Playwright Browsers
+        run: pnpm exec playwright install --with-deps
```

Since you added only Chrome in the playwright config, we can install only chromium to save a bit of time

```suggestion
        run: pnpm exec playwright install --with-deps chromium
```